pkg/template/builder.go

```diff
@@ -18,6 +20,23 @@ type Builder struct {
 	Functs template.FuncMap
 }
 
+// NewBuilder creates a builder with StaticCtx, LicenseCtx and ConfigCtx.
+func NewBuilder(configGroups []kotsv1beta1.ConfigGroup, existingValues map[string]ItemValue, localRegistry LocalRegistry, cipher *crypto.AESCipher, license *kotsv1beta1.License) (Builder, map[string]ItemValue, error) {
+	b := Builder{}
+	configCtx, err := b.newConfigContext(configGroups, existingValues, localRegistry, cipher, license)
+	if err != nil {
+		return Builder{}, nil, errors.Wrap(err, "create config context")
+	}
+
+	b.Ctx = []Ctx{
+		StaticCtx{},
+		LicenseCtx{License: license},
+		NewKurlContext("base", "default"), // can be hardcoded because kurl always deploys to the default namespace
```

Comment:
default is fine being hardcoded, not sure if 'base' (the name of that installer CRD object) is? dunno if we ever expect that to change